pyproject.toml

```diff
@@ -30,12 +30,12 @@ features such as type inference, gradual typing, generics and union
 types.
 """, content-type = "text/x-rst"}
 authors = [{name = "Jukka Lehtosalo", email = "jukka.lehtosalo@iki.fi"}]
-license = {text = "MIT"}
+license = "MIT"
+license-files = ["LICENSE", "mypy/typeshed/LICENSE"]
```

Comment:
Should `CREDITS` be included here? Until now it wasn't / is even explicitly excluded in `MANIFEST.in`.
https://github.com/python/mypy/blob/master/CREDITS

https://github.com/python/mypy/blob/765a78fee1e35e10e2c5a28c02f2bc73cc1aa4a9/MANIFEST.in#L48

Reply:
We haven't treated the credits file as part of the license in the past. It has only been updated very occasionally.